src/screens/registerProfessor/components/FormRegister/index.tsx

```diff
@@ -0,0 +1,143 @@
+import { Box, Typography, FormHelperText } from '@mui/material';
+import BottomArea from '../../../../components/bottomArea';
+import { TextField } from '../../../../components/textField';
+import { SCREENS } from '../../../../utils/screens';
+import useRegisterProfessor from '../hooks/useRegister';
+import {
+  Container,
+  StyledForm,
+  FormContainer,
+  TypographyRegister,
+  TypographyCredentials,
+  ContainerRegisterOption,
+  CancelRegisterLink,
+  RegisterButton,
+} from './styles';
+
+const FormRegister = () => {
+  const {
+    handleNameChange,
+    handleEmailChange,
+    handlePasswordChange,
+    handleConfPasswordChange,
+    handleLoginClick,
+    handleContinueClick,
+    name,
+    email,
+    password,
+    confPassword,
+    error,
+    isLoading,
+    nameError,
+    emailError,
+    passwordError,
+    confPasswordError,
+  } = useRegisterProfessor();
+
+  const NameError = () => {
+    if (!!nameError) return <FormHelperText error>{nameError}</FormHelperText>;
+
+    return <></>;
+  };
+
+  const EmailError = () => {
+    if (!!emailError)
+      return <FormHelperText error>{emailError}</FormHelperText>;
+
+    return <></>;
+  };
+
+  const PasswordError = () => {
+    if (!!passwordError)
+      return <FormHelperText error>{passwordError}</FormHelperText>;
+
+    return <></>;
+  };
+
+  const ConfPasswordError = () => {
+    if (!!confPasswordError)
+      return <FormHelperText error>{confPasswordError}</FormHelperText>;
+
+    return <></>;
+  };
+
+  return (
+    <Container>
+      <FormContainer>
+        <TypographyRegister>Cadastro de Professor</TypographyRegister>
```

Comment:
Coloca aqui como h4:
```suggestion
        <TypographyRegister variant="h4">Cadastro de Professor</TypographyRegister>
```

Para funcionar vc precisa adicionar isso aqui no theme dentro da chave `typography` tb:
```javascript
    h4: {
      fontWeight: 600,
      fontSize: 32,
      [`@media (max-width:${breakpoints.values.md}px)`]: {
        fontSize: '24px',
      },
    },
    h5: {
      fontWeight: 600,
      fontSize: 28,
      [`@media (max-width:${breakpoints.values.md}px)`]: {
        fontSize: '16px',
      },
    },
```